src/Configuration/ValueObject/Serializer/DatabaseConfigSerializer.php

```diff
@@ -30,6 +30,10 @@ public static function serialize(DatabaseConfig $databaseConfig): array
         if ($databaseConfig->hasSSLConnection()) {
             $config['ssl'] = SSLConnectionConfigSerializer::serialize($databaseConfig->getSslConnectionConfig());
         }
+
+        if ($databaseConfig->hasInitQueries()) {
```

Comment:
Tu napriklad potom nebude tato podmienka, ... `initQueries` budu v `$config` vzdy, ... minimalne `[]` prazdne pole -> bude to jednoduchsie.

Reply:
jj, může být 👍